samples/osgi-bundles/org.wso2.carbon.uuf.sample.simple-auth.bundle/src/main/java/org/wso2/carbon/uuf/sample/simpleauth/bundle/api/auth/PersistentSessionManager.java

```diff
@@ -0,0 +1,233 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
There is a extra space at the beginning of the line (after `*`). Let's remove it.